v3/projectd/ansi/65/projectd_65_ansi.json

```diff
@@ -0,0 +1,194 @@
+{
+  "name": "ProjectD 65% ANSI",
+  "vendorId": "0x3233",
+  "productId": "0x5319",
```

Comment:
This VID PID has already been used, please replace it with another one.

Reply:
@gksygithub Instead of creating a new folder, you need to make changes based on the projectd_outlaw_65_ansi keyboard.

Reply:
ok, From next time, I will follow.
Now this PR will get merged or Do I need to delete this? please let me know, thanks.

Reply:
@gksygithub You need to remove projectd_outlaw_65_ansi keyboard in this PR

Reply:
Your description says to remove the projectd_outlaw_65_ansi keyboard, but you didn't do that.

Reply:
Sorry, Deleted projectd_outlaw_65_ansi keyboard.

Reply:
![image](https://github.com/WestBerryVIA/keyboards/assets/89234190/4c6161e5-aace-414e-ae34-5a6a2e8da2d5)  
It's not deleted.